test/edu/stanford/dlss/was/TestWasapiDownloader.java

```diff
@@ -5,22 +5,26 @@
 import static org.junit.Assert.*;
 import org.junit.*;
 
-import edu.stanford.dlss.was.WasapiDownloader;
-
 public class TestWasapiDownloader {
 
-    @Before
-    public void setUp() {
-    }
+  @Before
+  public void setUp() {
+  }
```

Comment:
yeah, i'd just left these because they were there from the original skeleton code.  will remove.